WARNING: Dataset not found, nonexistent paths: ['E:\\360Downloads\\浏览器\\模糊神经网络\\demo1224\\data\\dataset\\val\\images\\mesh']
Downloading https://github.com/ultralytics/yolov5/releases/download/v1.0/coco128.zip ...
Overriding model.yaml nc=80 with nc=158
                 from  n    params  module                                  arguments
  0                -1  1      3520  models.common.Focus                     [3, 32, 3]
  1                -1  1     18560  models.common.Conv                      [32, 64, 3, 2]
  2                -1  1     18816  models.common.C3                        [64, 64, 1]
  3                -1  1     73984  models.common.Conv                      [64, 128, 3, 2]
  4                -1  1    156928  models.common.C3                        [128, 128, 3]
  5                -1  1    295424  models.common.Conv                      [128, 256, 3, 2]
  6                -1  1    625152  models.common.C3                        [256, 256, 3]
  7                -1  1   1180672  models.common.Conv                      [256, 512, 3, 2]
  8                -1  1    656896  models.common.SPP                       [512, 512, [5, 9, 13]]
  9                -1  1   1182720  models.common.C3                        [512, 512, 1, False]
 10                -1  1    131584  models.common.Conv                      [512, 256, 1, 1]
 11                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']
 12           [-1, 6]  1         0  models.common.Concat                    [1]
 13                -1  1    361984  models.common.C3                        [512, 256, 1, False]
 14                -1  1     33024  models.common.Conv                      [256, 128, 1, 1]
 15                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']
 16           [-1, 4]  1         0  models.common.Concat                    [1]
 17                -1  1     90880  models.common.C3                        [256, 128, 1, False]
 18                -1  1    147712  models.common.Conv                      [128, 128, 3, 2]
 19          [-1, 14]  1         0  models.common.Concat                    [1]
 20                -1  1    296448  models.common.C3                        [256, 256, 1, False]
 21                -1  1    590336  models.common.Conv                      [256, 256, 3, 2]
 22          [-1, 10]  1         0  models.common.Concat                    [1]
 23                -1  1   1182720  models.common.C3                        [512, 512, 1, False]
 24      [17, 20, 23]  1    439611  models.yolo.Detect                      [158, [[10, 13, 16, 30, 33, 23], [30, 61, 62, 45, 59, 119], [116, 90, 156, 198, 373, 326]], [128, 256, 512]]
Model Summary: 283 layers, 7486971 parameters, 7486971 gradients
Transferred 356/362 items from yolov5s.pt
Traceback (most recent call last):
  File "train.py", line 542, in <module>
    train(hyp, opt, device, tb_writer)
  File "train.py", line 99, in train
    check_dataset(data_dict)  # check
  File "E:\360Downloads\浏览器\模糊神经网络\demo1224\code\utils\general.py", line 234, in check_dataset
    torch.hub.download_url_to_file(s, f)
  File "C:\Users\Sun\AppData\Roaming\Python\Python37\site-packages\torch\hub.py", line 425, in download_url_to_file
    u = urlopen(req)
  File "D:\Python37\lib\urllib\request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "D:\Python37\lib\urllib\request.py", line 525, in open
    response = self._open(req, data)
  File "D:\Python37\lib\urllib\request.py", line 543, in _open
    '_open', req)
  File "D:\Python37\lib\urllib\request.py", line 503, in _call_chain
    result = func(*args)
  File "D:\Python37\lib\urllib\request.py", line 1360, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "D:\Python37\lib\urllib\request.py", line 1320, in do_open
    r = h.getresponse()
  File "D:\Python37\lib\http\client.py", line 1321, in getresponse
    response.begin()
  File "D:\Python37\lib\http\client.py", line 296, in begin
    version, status, reason = self._read_status()
  File "D:\Python37\lib\http\client.py", line 265, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response